pkg/arvo/lib/agentio.hoon

```diff
@@ -115,6 +115,15 @@
   ^-  card
   [%give %fact paths cage]
 ::
+++  fact-all
+  |=  =cage
+  ^-  card
+  =/  paths=(list path)
+    %+  turn  ~(tap by sup.bowl)
+    |=  [duct ship =path]
+    path
+  [%give %fact paths cage]
```

Comment:
If there are no subscribers, the `paths` here is `~`, but that has special semantics in gall, generally used for sending facts just to the subscription that was just opened. Suspect this causes the symptom described in #5138.